eblob_kit.py

```diff
@@ -408,15 +455,20 @@ def __len__(self):
 class Blob(object):
     """Abstraction to blob consisted from index and data files."""
 
-    def __init__(self, path, mode='rb'):
+    def __init__(self, path, mode='rb', is_minion_mode=False):
         """Initialize Blob object again @path."""
         if os.path.exists(path + '.index.sorted'):
             self._index_file = IndexFile(path + '.index.sorted', mode)
         elif os.path.exists(path + '.index'):
             self._index_file = IndexFile(path + '.index', mode)
         else:
             raise IOError('Could not find index for {}'.format(path))
+
         self._data_file = DataFile(path, mode)
+        self._eprint = (
```

Comment:
Let's move this logic out of `class Blob`, so it should accept already configured and correct logger.

Reply:
Redone with extra argument to `print_error`.

Reply:
`print_error` removed from code, all logging done within standard logging subsystem.